src/citation.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html>
+  <head>
+    <!-- NOTE: JUST REPLICATE THIS PAGE AND PUT THE REAL CONTENT IN pages FOLDER -->
+    <title>PAVICS</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" contents="PAVICS is a virtual laboratory facilitating the analysis of climate data. It provides access to several data collections ranging from observations, climate projections and reanalyses. It also provides a Python programming environment to analyze this data without the need to download it. This working environment is constantly updated with the most efficient libraries for climate data analysis, in addition to ensuring quality control on the provided data and associated metadata.">
```

Comment:
Probably not breaking anything, just for consistency, the french version do not have this description meta.

Reply:
@aulemahal indicates that this is probably only needed in the english index.html